WcaOnRails/app/webpacker/lib/requests/routes.js.erb

```diff
@@ -96,10 +96,24 @@ export const endDelegateProbationUrl = `<%= CGI.unescape(Rails.application.route
 
 // dummyRoleId is used temporarily because delegates is yet to get a role id
```

Comment:
I think this comment needs to be moved down to line 110.

Reply:
Oh yes, thanks for pointing.